addons/web_editor/static/src/scss/web_editor.frontend.scss

```diff
@@ -31,3 +31,21 @@
         }
     }
 }
+
+.o_we_shape {
+    @include o-position-absolute(0,0,0,0);
+    position: absolute !important;
```

Comment:
Why?

Reply:
because the background video class will set the position of all children in a section to relative and is more specific

Reply:
Meh. To improve later but at least explain that as a comment for now